script.js

```diff
@@ -75,9 +89,27 @@ const list = {
 const datatable = {
     view: "datatable",
     id: "table",
-    data: small_film_set,
+    url: "data/data.js",
     autoConfig: true,
-    scroll: "y"
+    scroll: "y",
+    select: true,
+    hover: "myhover",
+    on: {
+        onAfterSelect: dataToForm
+    },
+    columns: [
+        { id: "rank", header: "", width: 50, css: "gray" },
+        { id: "title", header: ["Film Title", { content: "textFilter" }], fillspace: true },
+        { id: "year", header: ["Released", { content: "numberFilter" }] },
+        { id: "votes", header: ["Votes", { content: "textFilter" }] },
+        { id: "rating", header: ["Rating", { content: "textFilter" }] },
+        { id: "deleteIcon", header: "", template: "{common.trashIcon()}" }
```

Comment:
Add sorting and filtering to all Datatable columns (as in the picture below)

there is no sorting for columns

Reply:
Didn't notice it. Fixed.